learning/tour-of-beam/frontend/lib/pages/tour/widgets/content.dart

```diff
@@ -52,12 +52,9 @@ class ContentWidget extends StatelessWidget {
             mainAxisAlignment: MainAxisAlignment.spaceBetween,
             children: [
               Expanded(
-                child: SingleChildScrollView(
-                  controller: ScrollController(),
```

Comment:
Controllers must never be created in build methods because build methods may be called at any time, and so the scrolling will be lost.

In `TourNotifier`, make a map of `unitId -> ScrollController` and then pick the controller in this widget with a method like `getOrCreateScrollController(String unitId)`.

Reply:
Do you mean the ScrollController that was inside of the deleted SingleChildScrollView?

Reply:
Yes. I mean, if you ever need a `ScrollController` for the content again, make it like that please.